### DISCUSSION TITLE
General Convolution Using Lax on GPU

### DISCUSSION THREAD

**Discussion by MLdoesML** [jax-ml/jax#6970] [Q&A] [answered]
Hi, can anyone explain what this error mean? What do they mean by unimplemented ? 

![Screenshot 2021-06-15 at 10 23 01 AM](https://user-images.githubusercontent.com/5188337/121983637-37e57200-cdc4-11eb-8242-f9d1adce1b01.png)

**Comment by jakevdp** [comment, marked as answer]
Hi – from the error, it looks like the GPU backend does not support convolutions for the data types you are using. If you can edit your question to show a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of code that reproduces the error (as pasted text rather than a screenshot, if possible) then we may be able to recommend an alternative.

**Reply by MLdoesML** [reply]
Ahhh got it thanks. I was wondering what the error meant as it says unimplemented.
